packages/vue/src/components/atoms/SfBadge/SfBadge.vue

```diff
@@ -1,5 +1,10 @@
-<template>
-  <div class="sf-badge">
+<template functional>
+  <div
+    :class="[data.class, data.staticClass, 'sf-badge']"
+    :style="[data.style, data.staticStyle]"
+    v-bind="data.attrs"
```

Comment:
once we use `v-bind`, I don't think we need to specific `style = data.style`. Can you verify?

Reply:
Unfortunately class and style attributes are not a part of data.attrs in functional components so I need to add them separately.